store/tools/go.mod

```diff
@@ -0,0 +1,31 @@
+module github.com/confio/ics23-iavl
```

Comment:
```suggestion
module github.com/cosmos/cosmos-sdk/store/tools
```
im guess we would cut v1 for this after its merged? @robert-zaremba

Reply:
yes